library/alloc/src/ffi/c_str.rs

```diff
@@ -716,6 +716,12 @@ impl fmt::Debug for CString {
     }
 }
 
+impl fmt::Display for CString {
```

Comment:
This is missing a `#[stable]` attribute. Note that `#[unstable]` doesn't work on trait implementations.

Reply:
Done.